tools/extra/pac/fpgabist/dma/fpga_dma_test.c

```diff
@@ -92,7 +92,7 @@ void help(void)
 /*
  * Parse command line arguments
  */
-#define GETOPT_STRING ":hVNB:D:F:S:AIQ"
+#define GETOPT_STRING "B"
```

Comment:
I think you need a colon after the B
![image](https://user-images.githubusercontent.com/22808196/42060788-d10a44c6-7adc-11e8-8739-4e12d29ab689.png)